packages/core/stories/tokens/typography/Typography.stories.mdx

```diff
@@ -23,146 +23,181 @@ import generateTitle from '../../util/generateTitle';
   }}
 />
 
-# Noto Sans
-**The official typeface of Mayflower is Noto Sans.**
-It’s carefully chosen for the following reasons:
-- Accessible and easy to read
-- To enable all Commonwealth agencies to design, produce and deliver online and offline content with consistent typography, without additional costs or legal barriers.
-- To provide comprehensive language coverage for content internationalization.
-- To support the guiding principles of the Mass.gov brand identities: modern, dignified, human, helpful
+# Typography
+
+
+## The official typeface of Mayflower is **Noto Sans**
+
+Noto Sans was chosen because it's...
+- **Accessible** and easy-to-read for users with all levels of language proficiency
+- **Free** and Open-source so agencies can produce print and digital design, without barriers
+- **Multilingual** to cover the languages our constituents speak and read
+- **Modern** and consistent with the principles of the Mass.gov brand
 
 Using these fonts for your print and digital assets is a subtle but compelling way to create a voice that is cohesive and unified with the rest of the Commonwealth digital ecosystem, and to ensure the text is accessible and easy to read.
 
 > Noto Sans is an open source typeface jointly designed and developed by Google and Monotype. It has the most comprehensive language coverage — including more than 800 languages and 100 written scripts. You can directly download the fonts from Google Font, or for web projects, link to the fonts via Mayflower Assets CDN
 
-
 ## Characters
-<RichText className="sb-block">
-  <div className="sb-text">{letters.toUpperCase()}</div>
-  <div className="sb-text">{letters}</div>
-  <hr />
-  <div className="sb-text">{numbers}</div>
-  <div className="sb-text">{numbersReverse}</div>
-  <hr />
-  <div className="sb-text">{characters}</div>
-</RichText>
 
+<div>
+	<p className="font-specimen large">{letters.toUpperCase()}<br />{letters}</p>
```

Comment:
I want to also display lower case letters and numbers here. Is there a reason those are removed?

Reply:
Good point. I thought it was redundant since they are in the block of all characters, but it is nice to have them with the alphabet.

Reply:
I put them back.